.github/workflows/dump-ci-stats.yml

```diff
@@ -0,0 +1,39 @@
+name: Dump GH CI Stats
+
+on:
+  workflow_run: # This allows the workflow to be reused
+    workflows:
+      [
+        'Integration Tests',
+        'Test Golang',
+        'golangci-lint',
+        'Build release Docker Images',
+        'Test all Packages',
+        'Test Documentation',
+        'Manage integration check',
+        'after-merge.yml',
+      ]
+    types:
+      - completed
+
+jobs:
+  dump_ci_stats:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install GCP Monitoring/Metrics Client
+        run: yarn add @google-cloud/monitoring --ignore-workspace-root-check
```

Comment:
minor improvement:
We can cache it. "cache: yarn" can help.
Will do it as a follow-up together with othetr tweakings.